extensions/ql-vscode/src/databases.ts

```diff
@@ -542,6 +545,24 @@ export class DatabaseManager extends DisposableObject {
     return databaseItem;
   }
 
+  private onQueryServerRestarted = async (
```

Comment:
Suggest calling this `reregisterDatabases` and adding an explanatory comment. (Once it gets passed to `qs`, that code knows it will be run on restart.)